Makefile

```diff
@@ -40,9 +40,10 @@ docker.test: docker.images
 war: ${WAR}
 war.publish: ${WAR}
 	ssh ${SSH_OPTS} ${PKGSERVER} mkdir -p "'${WARDIR}/${VERSION}/'"
-	sha256sum ${WAR} > ${WAR_SHASUM}
+	rsync -avz -e "ssh ${SSH_OPTS}" "${WAR}" "${PKGSERVER}:${WARDIR}/${VERSION}/${ARTIFACTNAME}.war"
+	sha256sum ${WAR} | sed 's, .*/, ,' > ${WAR_SHASUM}
```

Comment:
Is sha256 installed by the packaging dockerfile and called out in the dependencies for build (README and setup?)